Core/Src/monitor.c

```diff
@@ -183,6 +185,18 @@ void vPedalsMonitor(void* pv_params)
 			fault_data.diag = "Failed to push pedal data to queue";
 			queue_fault(&fault_data);
 		}
+
+		/* Send CAN messages with raw pedal readings, we do not care if it fails*/
+		endian_swap(&adc_data[ACCELPIN_1], sizeof(adc_data[ACCELPIN_1]));
+		endian_swap(&adc_data[ACCELPIN_2], sizeof(adc_data[ACCELPIN_2]));
+		memcpy(accel_pedals_msg.data, &adc_data, accel_pedals_msg.len);
+		queue_can_msg(accel_pedals_msg);
```

Comment:
So pedal position broadcast over CAN prob will get very bad with this implementation, specifically because of the rate at which we sample the pedals vs the rate at which we send CAN messages. I'd recommend downsampling the pedal position to be broadcast (i.e. only send every 5th pedal reading) such that we don't clog CAN dispatch

Reply:
Will do.

Reply:
Did simple counter.  if you want a timer lmk.